### ISSUE TITLE
Lesion over bony prominences

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9913]
**Preferred term label:**
Lesion over bony prominences

**Synonyms**
Heel, knee, elbow, epicondyle, malleolus

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Clavus durum

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000301
- label: Localized over bony prominences
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated on the skin in areas where bones are close to the surface (bony prominences).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9913
- synonyms:
